assets/js/src/form_editor/blocks/input_styles_settings.jsx

```diff
@@ -1,16 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
 import MailPoet from 'mailpoet';
 import {
   Panel,
   PanelBody,
+  ToggleControl,
 } from '@wordpress/components';
+import { partial } from 'underscore';
```

Comment:
why underscore? I thought we are going to slowly move to `lodash` and remove underscore eventually.